lib/__tests__/emailnotifications_queue_handler.test.ts

```diff
@@ -167,7 +167,9 @@ describe("handleNotification", () => {
     expect(sentMail.data.headers["X-Italia-Messages-NotificationId"]).toBe(
       "A_NOTIFICATION_ID"
     );
-    expect(sentMail.data.html).toBe("<h1>Hello world!</h1>");
+    expect(
+      String(sentMail.data.html).indexOf("<h1>Hello world!</h1>")
+    ).toBeGreaterThan(0);
```

Comment:
maybe it's greater or equal ?

Reply:
Greater because there's supposed to be a `document` section before, the HTML shouldn't start with `<h1>`